en-US/Language.xml

```diff
@@ -1054,7 +1068,14 @@ Numbers (move from N entry) -->
 								</row>
 								 <row>
 									<entry> To create another administrator, click <guimenuitem>New</guimenuitem> on the <guimenuitem>File</guimenuitem> menu. </entry>
```

Comment:
A little confused, this seems identical to line 1073, not sure what I'm mnissing.

Reply:
I assume that the distinction here is between administrator as an account versus as a person.